hrms/hr/doctype/leave_block_list/leave_block_list.js

```diff
@@ -3,6 +3,101 @@
 
 frappe.ui.form.on('Leave Block List', {
 	refresh: function(frm) {
+		frm.add_custom_button(__("Bulk Add Dates"), function () {
```

Comment:
Add this button as a field above the dates table instead, since this action is associated with adding dates to the **Leave Block List Dates** table. 

Other forms where this pattern is used:
<img width="1295" alt="image" src="https://user-images.githubusercontent.com/24353136/199908174-c68be3a8-b6c8-4525-9e1d-980494705aba.png">
 
<img width="1295" alt="image" src="https://user-images.githubusercontent.com/24353136/199908517-5cf6d73a-db32-4263-9e50-1b607fa262b5.png">

Reply:
Done, changed the label to `Add Day-wise Dates`